server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,67 @@
+import { Review } from "../../models/index.js"
+
+class ReviewSeeder {
+  static async seed() {
+    const reviewData = [
+      {
+        userId: 1,
+        burgerId: 1,
```

Comment:
we should try to avoid hardcoding `id`'s that reference other objects, but instead query for those objects in the seeder file here to make sure they exist